pages/hosted-lead.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: page-fullwidth
+show_meta: false
+title: "Lead Instructor checklist"
+header:
+    image_fullwidth: "wood_plank.jpg"
+permalink: "/hosted-lead/"
+---
+
+**Information on lead instructor responsibilities for centrally-organized workshops.**
+
+Lead instructors are responsible for co-ordinating with co-instructors and the host about the curriculum. They are also responsible for setting up the workshop's web page and for distributing learner surveys. All of these duties can be delegated, but it's the lead instructor's responsibility to make sure they are done. If you've been assigned the lead instructor role, but aren't comfortable with it, or find that you don't have the time, let the [Data Carpentry workshop coordinator](mailto:admin@datacarpentry.org) know as soon as possible (admin@datacarpentry.org).
+
+
+The lead instructor is also responsible for preparing to teach the material - see the [Instructor checklist](/instructor-checklist/) for details.
+
+*If your workshop is self-organized, the lead instructor is often also the host and is responsible for <u>all</u> logistics. See the [self-organized host/lead-instructor checklist](/self-org-lead/) for hosting responsibilities for self-organized workshops.*
+
+## Before the workshop:  
+- [ ] With the workshop host and your co-instructors, decide on your [curriculum](/workshops/), the order of lessons and who will teach each lesson  
+- [ ] Create an Etherpad for your workshop at http://pad.software-carpentry.org/YYYY-MM-DD-site (where "YYYY-MM-DD-site" is the ID of your workshop)
+- [ ] [Set up your workshop website](https://github.com/datacarpentry/workshop-template)  
+- [ ] Send your website repository’s URL to the [Data Carpentry workshop coordinator](mailto:admin@datacarpentry.org) (admin@datacarpentry.org)  
+- [ ] Assist the workshop host with [recruiting helpers](/email-templates/#recruiting-helpers) (at least 1 per 12 participants)  
+- [ ] Coordinate with the host to ensure the workshop is [accessible](/accessibility/)  
```

Comment:
[  ] Check with the host that they have the necessary [equipment](/equipment-checklist/), especially sticky-notes